### ISSUE TITLE
Matrix Initialisation Empty

### ISSUE THREAD

**Issue by aanchalm12** [BurksEngineering/TcMatrix#2]
Hi, 

I am using this library to perform matrix functions. It's working great except when I am using the following code.
vMatrix.IsEmpty is true. vMatrix.Rows = 3. vMatrix.Cols = 0. 
This looks similar to the quick start example given.  

VAR
  v: ARRAY[1..3, 1..1] OF LREAL;
  vMatrix : Array2DStaticMatrix;
END_VAR

v[1, 1] := 1;
v[2, 1] := 2;
v[3, 1] := 3;

vMatrix(Data := v);

**Comment by BurksEngineering** [comment]
Thanks for letting me know! I just added the Array2DStaticMatrix to make the code more accessible, but it looks like my dinky little test didn't catch this glaring issue. I just added some more conditions to the test, it recreated the issue, and I found the source of the problem in some helper function.

Check out release 1.4.1 which has a fix for this issue. Let me know if you run in to anything else!
